lib/pages/authenticate/edit_account.dart

```diff
@@ -27,13 +35,28 @@ class EditAccountPageState extends State<EditAccountPage> {
   final _confirmPasswordController = TextEditingController();
   late final Auth authentitcationHelper;
   late FirebaseAuth auth;
+  late FirebaseFirestore firebaseFirestore;
 
-  EditAccountPageState(this.auth);
+  EditAccountPageState(this.auth, this.firebaseFirestore);
 
   @override
   void initState() {
     super.initState();
-    authentitcationHelper = Auth(auth: auth);
+    authentitcationHelper = Auth(auth: auth, firestore: firebaseFirestore);
+  }
+
+  // Displays an alert dialog with the text passed as parameter.
+  void show_alert_dialog(String text) {
```

Comment:
delete this method (this probably indicates that merge conflicts were not solved the right way because I did remove that one. Please make sure that all the places that show an alert dialog use the helper class I built (error_dialog_helper.dart)